resources/css/bem/wiki-search.less

```diff
@@ -12,15 +12,19 @@
   position: relative;
 
   &__bar {
-    .default-border-radius();
     display: flex;
     flex-direction: row;
 
     padding: 13px @_gutter;
     font-size: @font-size--wiki-search;
     background-color: @osu-colour-b6;
 
-    border: @_border-size solid @osu-colour-orange-1;
+    border: 1px solid @osu-colour-b6;
```

Comment:
update `@_border-size` instead otherwise the suggestion box would have wrong alignment

also don't use `@osu-colour-` anymore. New codes should always use the css variable. Adding more lines that need update isn't helping.